res/css/views/rooms/_EventTile.scss

```diff
@@ -18,233 +18,379 @@ limitations under the License.
 $left-gutter: 64px;
 $hover-select-border: 4px;
 
-.mx_EventTile {
+.mx_EventTile:not([data-layout=bubble]) {
```

Comment:
The diff in that file is slightly horrendous to look at.

The bottom line is to exclude a lot of styles that are applied for the `Group` and `IRC` layout to avoid doing many overrides in the `Bubble` layout`.

And everything that falls below this big `.mx_EventTile:not([data-layout=bubble])` block are things that will remain the same regardless of the layout (the e2ee shields, avatar shapes, ...)

Very little code has been changed, it's mainly been reshuffled